.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,170 @@
+
+#### Root
+
+- `/flow-typed` – Flow library definitions
+- `/scripts` – Scripts for development and publishing
+
+### How to develop
+
+Install dependencies:
+
+```sh
+yarn install
+```
+
+Build the design tokens and the tailwind preset:
+
+```sh
+yarn tokens build
+yarn tailwind-preset build
+```
+
+For developing Orbit, components run Storybook:
+
+```sh
+yarn dev
+```
+
+If you would like to try out the development version of one of our packages, we recommend using [yalc](https://github.com/wclr/yalc):
+
+```sh
+yarn global add yalc
+```
+
+This is how you would link `@kiwicom/orbit-components`:
+
+```sh
+cd packages/orbit-components
+yalc publish
+cd path/to/your/project
+yalc add @kiwicom/orbit-components
+```
+
+Then run the following every time you make new changes in `packages/orbit-components` that you want to try out:
+
+```sh
+yalc push
+```
+
+This is a shortcut for `yalc publish --push`, which "publishes" the package again and updates it in every project that installed it.
+
+### Testing
+
+We encourage adding tests to all components in Orbit. Jest is the testing framework across all types of tests in Orbit.
+
+> Before writing the test for a new component, first check [component testing conventions](contribution/testing-conventions).
+
+Types of tests required to pass on every component:
+
+- **Unit tests**
+- **Visual regression test**
+
+### Unit tests
+
+For checking the correctness of the component run `yarn test-ci` to perform the same automatic checks as the CI server. You can also run these checks separately by typing:
+
+- `yarn check:types` - TypeScript check
+- `yarn eslint` for ESLint check
+  - some warnings can be fixed automatically by running `yarn eslint --fix`
+- `yarn test` to run all Jest tests
+
+If you need to update tests because of some changes in the code, you can do it by running `yarn test -u`. We strongly recommend testing your component in different types of browsers across different platforms.
+
+### Visual regression test
+
+See our testing conventions for more information about [visual regression tests](contribution/testing-conventions).
+
+## Contributing to the docs
+
+See our full guide to [contributing to our docs](contribution/docs)
+
+## Adding a new icon
+
+It's easy to update or add a new icon because SVGs are automatically converted to React components. See [this documentation](contribution/icons) for more information.
+
+## Props naming convention
+
+We want to ensure that all props are consistent across all components. [This convention](contribution/props-convention) can help choose the right prop names in for your new components.
+
+## Component design
+
+Check [this section](contribution/component-design) for a complete description of the structure of the files which are necessary for a Component check.
+
+## Commits
+
+We are using [this commits convention](contribution/commits) to ensure that all commits are pushed in some logic from all devs.
```

Comment:
I don't think "in some logic from all devs" is what we want to say here. Can you rephrase it? 🙏

Reply:
Definitely 😅 I copied it blindly from the other doc.